plenum/test/primary_selection/test_primary_selection_routes.py

```diff
@@ -0,0 +1,7 @@
+from plenum.test.conftest import looper
+
+nodeCount = 7
+
+
+def test_routes(nodeSet, up):
+    pass
```

Comment:
Since all tests work, we have this already tested, but separate unit tests can come separately